tsconfig.json

```diff
@@ -96,7 +96,8 @@
   },
   "exclude": [
     "node_modules",
-    "vendor"
+    "vendor",
+    "dist",
```

Comment:
Exclude it from what? Compilation?

Reply:
https://www.typescriptlang.org/tsconfig/#exclude